carball/analysis/stats/boost/boost.py

```diff
@@ -86,13 +95,13 @@ def get_time_with_zero_boost(data_frame: pd.DataFrame, player_dataframe: pd.Data
 
     @staticmethod
     def get_player_boost_collection(player_dataframe: pd.DataFrame) -> Dict[str, int]:
-        value_counts = player_dataframe.boost_collect.value_counts()
         try:
+            value_counts = player_dataframe.boost_collect.value_counts()
             return {
                 'big': int(value_counts[True]),
                 'small': int(value_counts[False])
             }
-        except KeyError:
+        except (AttributeError, KeyError):
```

Comment:
Why would there be an AttributeError? boost_collect is None?

Reply:
if the column does not exist